api/config/v1alpha1/helpers.go

```diff
@@ -51,6 +53,36 @@ func DefaultGateway() *Gateway {
 	}
 }
 
+// DefaultEnvoyGatewayLogging returns a new EnvoyGatewayLogging with default configuration parameters.
+func DefaultEnvoyGatewayLogging() *EnvoyGatewayLogging {
+	return &EnvoyGatewayLogging{
+		Level: map[EnvoyGatewayLogComponent]LogLevel{
+			LogComponentGatewayDefault: LogLevelInfo,
+		},
+	}
+}
+
+// DefaultEnvoyGatewayLoggingLevel returns a new EnvoyGatewayLogging with default configuration parameters.
+// When v1alpha1.LogComponentGatewayDefault specified, all other logging components are ignored except "gateway-api" which is set to "warn".
+func DefaultEnvoyGatewayLoggingLevel(component string, level LogLevel) LogLevel {
+	if string(LogComponentGatewayApiRunner) == component && level == LogLevelWarn {
```

Comment:
the gateway-api logic here need to be removed

Reply:
for your meeting feedback

- to enable debug logging for all components except for gateway-api which is set to warn

so, change to use `default` enable for all?

Reply:
Yah

Reply:
change it?